tabulator/parsers/ndjson.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import jsonlines
+
+from .. import helpers
+from . import api
+
+
+# Module API
+
+class NDJSONParser(api.Parser):
+    """Parser to parse NDJSON data format.
+
+    See: http://specs.okfnlabs.org/ndjson/
+    """
+
+    # Public
+
+    options = [
+    ]
+
+    # XXX: I would suggest to move this whole boilerplate to api.Parser, most
```

Comment:
The reason behind it is intention to avoid inheritance instead of interface (`parser.API`) implementation.